.travis.yml

```diff
@@ -62,6 +62,16 @@ jobs:
         PIP_DEPENDENCIES=.
         TEST_COMMAND=verify_install_requires
 
+    - name: Build distribution
+      env:
+        PIP_DEPENDENCIES=pep517
+        TEST_COMMAND="python -m pep517.build ."
+
+    - name: Security check
+      env:
+        PIP_DEPENDENCIES=".[test] bandit"
+        TEST_COMMAND="bandit --exclude 'jwst/*test*' -r jwst scripts"
```

Comment:
The `exclude` flag is not picked up, perhaps the name is different.
I've done the bandit configuration in gwcs/.bandit.yaml and it works.
I suggest the same configuration is done for jwst.
Most (if not all) of the assert errors I see are in the test files. When those are cleaned
we can see if there are anything remaining to fix.